roles/confluent.variables/vars/main.yml

```diff
@@ -1,6 +1,9 @@
 ---
 confluent_ansible_branch: 6.1.0-post
 
+control_center_metrics_delegate_kafka_broker_hosts: "{{ groups[control_center_metrics_kafka_broker_ansible_group] | default(['localhost']) }}"
+control_center_metrics_delegate_kafka_broker_hostvars: "{{ hostvars[control_center_metrics_delegate_kafka_broker_hosts | first] }}"
```

Comment:
not sure about these variable names.... what are some other ideas?
metrics_reporter_kafka_broker_hosts
metrics_reporter_kafka_broker_hostvars

also maybe
metrics_reporter_kafka_broker_ansible_group

I know you wanted to move away from telemetry, but the feature is called metrics reporter right?

Reply:
I chose not to use "reporter" or "interceptor" in the name due to potential for confusion with the individual settings. I chose Control Center, since this all has to do with Control Center Metrics and to also separate this all from the new Telemetry metrics.

Reply:
i see your point about metrics reporter and monitoring interceptors having a name conflict... 

whats the significance of delegate?
maybe removing delegate would help. wdyt?

Reply:
Delegate, was to show that we are leveraging a single kafka node of the group as a stand in for the whole cluster. Which is why it also only appears in the `vars` and not the `defaults` collection.